pkg/monitor/controller.go

```diff
@@ -173,6 +220,34 @@ func (monitor *controllerMonitor) deleteStaleAgentCRD(old interface{}) {
 	monitor.deleteAgentCRD(node.Name)
 }
 
+func (monitor *controllerMonitor) createExternalNodeAgentCRD(obj interface{}) {
+	en := obj.(*v1alpha1.ExternalNode)
+	klog.Infof("Creating ExternalNode agent monitoring CRD %s", en.Name)
+	agentCRD := new(v1beta1.AntreaAgentInfo)
+	agentCRD.Name = en.Name
+	_, err := monitor.client.CrdV1beta1().AntreaAgentInfos().Create(context.TODO(), agentCRD, metav1.CreateOptions{})
```

Comment:
We should check and ignore the case the AgentInfo CRD is already created for this Node.

Reply:
Thanks for pointing out. I have updated this part.